bundles/best_practices.rst

```diff
@@ -476,3 +539,4 @@ Learn more
 .. _`Packagist`: https://packagist.org/
 .. _`choose any license`: http://choosealicense.com/
 .. _`valid license identifier`: https://spdx.org/licenses/
+.. _`Travis CI`: travis-ci.org
```

Comment:
use `https://travis-ci.org` (btw, I think the current link would be broken, as it would be a relative link, and so would stay on symfony.com)